lib/_tls_wrap.js

```diff
@@ -560,6 +561,11 @@ TLSSocket.prototype._init = function(socket, wrap) {
 };
 
 TLSSocket.prototype.renegotiate = function(options, callback) {
+  if (options === null || typeof options !== 'object')
+    throw new ERR_INVALID_ARG_TYPE('options', 'Object', options);
+  if (callback != null && typeof callback !== 'function')
```

Comment:
Do we need to support `null` here? I think just `undefined` would be enough.

Reply:
Sorry @cjihrig, I just saw the approvaled state, not this comment. I see your point, should I open another PR to change this?

Reply:
I think it's your call. It's not a huge deal, but I do think `null` implies a mistake or different intention from `undefined`.